frontend/src/app/Settings/Users/ManageUsers.scss

```diff
@@ -6,6 +6,7 @@
   .users-sidenav {
     border-right: 1px solid #dfe1e2;
     max-width: 25%;
+    min-width: 25%;
```

Comment:
I think we can just set `width : 25%` here instead of specifying both upper and lower bound.